core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1699,4 +1710,92 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 
 		return $label;
 	}
+
+	/**
+	 * @param array<int|string> $ids
+	 */
+	protected function preloadCurrentRecords(array $ids, string $table): void
+	{
+		// Clear current cache to make sure records are gone if they cannot be loaded from the database below
+		foreach ($ids as $id)
+		{
+			$this->setCurrentRecordCache($id, $table, null);
+		}
+
+		/** @var Connection $connection */
+		$connection = System::getContainer()->get('database_connection');
+
+		$stmt = $connection->executeQuery(
+			'SELECT * FROM ' . $table . ' WHERE id IN (?)',
```

Comment:
```suggestion
			'SELECT * FROM ' . $connection->quoteIdentifier($table) . ' WHERE id IN (?)',
```

Reply:
we don't do that **anywhere** in the data containers

Reply:
We currently don’t quote any table name in DC_Table, so I think this is not needed here.

Reply:
We do in L1469 for instance. But I'm fine anyway.